#! /bin/sh /usr/share/dpatch/dpatch-run
## fix-retrieve-op-panel-cfg.dpatch by Diego Iastrubni <diego.iastrubni@xorcom.com>
##
## DP: By default freePBX's script saves into the webroot. it should save the config into /etc/op-panel/freepbx

@DPATCH@
diff -urNad freepbx-2.2.0~dfsg~/amp_conf/bin/retrieve_op_conf_from_mysql.pl freepbx-2.2.0~dfsg/amp_conf/bin/retrieve_op_conf_from_mysql.pl
--- freepbx-2.2.0~dfsg~/amp_conf/bin/retrieve_op_conf_from_mysql.pl	2007-02-26 10:29:49.000000000 +0000
+++ freepbx-2.2.0~dfsg/amp_conf/bin/retrieve_op_conf_from_mysql.pl	2007-02-26 10:34:23.000000000 +0000
@@ -138,7 +138,7 @@
 $ampconf = parse_amportal_conf( "/etc/amportal.conf" );
 
 # WARNING: this file will be substituted by the output of this program
-$op_conf = $ampconf->{"AMPWEBROOT"}."/panel/op_buttons_additional.cfg";
+$op_conf = "/etc/op-panel/freepbx/op_buttons_additional.cfg";
 # username to connect to the database
 $username = $ampconf->{"AMPDBUSER"};
 # password to connect to the database
diff -urNad freepbx-2.2.0~dfsg~/amp_conf/htdocs_panel/op_buttons.cfg freepbx-2.2.0~dfsg/amp_conf/htdocs_panel/op_buttons.cfg
--- freepbx-2.2.0~dfsg~/amp_conf/htdocs_panel/op_buttons.cfg	2006-10-16 01:57:56.000000000 +0100
+++ freepbx-2.2.0~dfsg/amp_conf/htdocs_panel/op_buttons.cfg	2007-02-26 10:36:17.000000000 +0000
@@ -1,7 +1,7 @@
-include => op_buttons_additional.cfg
+include => freepbx/op_buttons_additional.cfg
 
 ; use this for your customizations to FOP
-include => op_buttons_custom.cfg
+include => freepbx/op_buttons_custom.cfg
 
 ; queues
 [rectangle]
